akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -207,12 +207,13 @@ object ActorMaterializerSettings {
     subscriptionTimeoutSettings: StreamSubscriptionTimeoutSettings,
     debugLogging: Boolean,
     outputBurstLimit: Int,
+    syncProcessingLimit: Int,
```

Comment:
After switching on MiMa you’ll see that the old method signature must be kept, you cannot modify it; the only possibility is to add another method overload, but we might just not do that and rely on reading from config + withXXX methods for new properties.

But this point is going to become moot once the event limit is reused to control the fast path as well.